AnswerDotAI/fasthtml
# Support iterators as view arguments

This resolves #63. To test this out:

1. Go to the examples directory
2. Run `python adv_app.py`
3. Reorder tasks, they should no longer fail
